BabylonJS/Babylon.js
# Playwright visualization tests

This PR adds a new script to run playwright visualization tests.

To test playwright:

1. run `npm install` 
2. run `npx playwright install` (this will install the missing browsers)
3. run `npm start` (or start the babylon-server any other way)
4. run `npm run test:playwright -w @tools/tests -- --project=webgl2 --ui` to start webgl2 tests in UI mode

This will start playwright in UI mode. You can run tests individually and see the results or run all of them:

![image](https://github.com/BabylonJS/Babylon.js/assets/1381702/ad4f040b-7800-4a2e-b4f3-cfa8ebc8180f)

other projects - webgl1 and webgpu

For automated tests run

`npm run test:playwright -w @tools/tests -- --project=webgl2` . This will run headless with no UI. If something fails playwright will open its HTML reporter to show you exactly what happened.

Alternative to 3 (to test against public CDN), set the CDN_BASE_URL environment variable to "https://cdn.babylonjs.com".
